docs/src/docs/asciidoc/user/clustered-cache.adoc

```diff
@@ -134,7 +134,7 @@ It contains the bare minimum configuration required for the samples in the rest
 
 [source,xml,indent=0]
 ----
-include::{sourcedir38}/clustered/client/src/test/resources/configs/docs/tc-config.xml[]
+include::{sourcedir38}/clustered/client/src/test/resources/configs/docs/cluster.properties[]
 ----
 
 The above configuration defines two named server off-heap resources:
```

Comment:
The callouts at lines 142 and 143 are now dead. You'd need to define callouts in the properties file.

IMO, we don't need a properties file merely to mention `offheap-resources`. I'd include the lone `offheap-resources=primary-server-resource:128MB,secondary-server-resource:96MB` line here, and get rid of the config file. Note that unlike before, having a config file isn't mandatory now - you could specify `offheap-resources` during server startup without needing a config file.